src/SessionClient.ts

```diff
@@ -0,0 +1,260 @@
+     * @return {Promise<object,any>} The session result object with details on what was submitted
+     * @see https://developers.nexosis.com/docs/services/98847a3fbbe64f73aa959d3cededb3af/operations/59149d7da730020f20dd41a8
+     */
+    get(id: string) {
+        return this._apiConnection.get(`sessions/${id}`, this.FetchTransformFunction);
+    }
+
+    /**
+     * Get status header only of a session by sessionId
+     * 
+     * @param {string} id - a session id returned from a previous request to start a session. 
+     * @return {Promise<object,any>} The session header with status
+     * @see https://developers.nexosis.com/docs/services/98847a3fbbe64f73aa959d3cededb3af/operations/59149d7da730020f20dd41a9
+     */
+    status(id: string) {
+        return this._apiConnection.head(`sessions/${id}`, this.FetchTransformFunction).then(headers => {
+            return headers.get('nexosis-session-status');
+        });
+    }
+
+    /**
+     * Remove a session and its results from your account
+     * 
+     * @param {string} id - a session id returned from a previous request to start a session. 
+     * @see https://developers.nexosis.com/docs/services/98847a3fbbe64f73aa959d3cededb3af/operations/593949c4e0ef6e0cb481aa31
+     */
+    remove(id: string) {
+        return this._apiConnection.delete(`sessions/${id}`, this.FetchTransformFunction);
+    }
+
+    /**
+     * Get the results of a session
+     * 
+     * @param {string} id - a session id returned from a previous request to start a session. 
+     * @return {Promise<object,any>} The session with all result rows
+     * @see https://developers.nexosis.com/docs/services/98847a3fbbe64f73aa959d3cededb3af/operations/59149d7da730020f20dd41a7
+     */
+    results(id: string) {
+        return this._apiConnection.get(`sessions/${id}/results`, this.FetchTransformFunction);
+    }
+
+    /**
+     * Get the specific interval results of a session
+     * 
+     * @param {string} id - a session id returned from a previous request to start a session. 
+     * @param {string} predictionInterval - a specific interval from the session's availablePredictionIntervals
+     * @return {Promise<object,any>} The session with all result rows
+     * @see https://developers.nexosis.com/docs/services/98847a3fbbe64f73aa959d3cededb3af/operations/59149d7da730020f20dd41a7
+     */
+    resultsByInterval(id: string, predictionInterval: number) {
+        return this._apiConnection.get(`sessions/${id}/results`, this.FetchTransformFunction, { "predictionInterval": predictionInterval });
+    }
+
+    /**
+     * 
+     * @param {string} id - completed classification model building id
+     * @return {Promise<object,any} The results with 'confusionMatrix' and 'classes' properties 
+     */
+    confusionMatrixResults(id: string) {
+        return this._apiConnection.get(`sessions/${id}/results/confusionmatrix`, this.FetchTransformFunction);
+    }
+
```

Comment:
Not sure when it went in versus the confusion matrix endpoint, but this is missing the class scores (/sessions/{sessionId}/results/classScores) endpoint.